nalgebra-sparse/src/csc.rs

```diff
@@ -720,12 +720,14 @@ macro_rules! impl_csc_col_common_methods {
                 self.lane.get_entry(global_row_index)
             }
 
-            /// Iterator over the row indices and elements of a column of a CSC matrix.
+            /// Iterator over `(row_idx, value)`, the row indices and elements of a column of a CSC matrix.
             /// Equivalent to `col.row_indices().iter().zip(col.values().iter())`.
```

Comment:
Can you perhaps manually check the rendering for the docs for the new functionality by running `cargo doc`? I think you need a line break after the first sentence here, otherwise you'll get the second line in the "summary", which bloats the function overview for the module.

Same applies for `iter_mut` and CSR variants etc.!